fluent/migrate/context.py

```diff
@@ -139,7 +139,7 @@ def maybe_add_localization(self, path):
         else:
             self.localization_resources[path] = collection
 
-    def add_transforms(self, path, reference, transforms):
+    def add_transforms(self, target, reference, transforms):
         """Define transforms for path using reference as template.
```

Comment:
Want to adjust the comment to change from path to target, too?

Reply:
Sure, I considered it but thought that the current comment was still OK. Happy to change it.

Reply:
I think an udpate would be good, as right now, `for path` isn't good English, if path isn't the argument.

Reply:
Yeah, agreed. I updated the comment and also added two sentences about what those paths are supposed to be relative to.